src/core/dev_api/openvino/core/tensor_util.hpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/op/constant.hpp"
+
+namespace ov {
+namespace util {
+
+OPENVINO_API Tensor ge(const ov::Tensor& lhs, const ov::Tensor& rhs);
+template <typename T>
+OPENVINO_API Tensor ge(const ov::Tensor& lhs, const T& element);
+OPENVINO_API bool all(const ov::Tensor& t);
+template <typename T>
+OPENVINO_API std::vector<T> to_vector(const ov::Tensor& t);
+
+template <typename T>
+Tensor make_tensor_of_value(const element::Type_t& et, const Shape& shape, const std::vector<T>& value) {
+    auto c = op::v0::Constant(et, shape, value);
+    auto t = Tensor(et, shape);
+    std::memcpy(t.data(), c.get_data_ptr(), t.get_byte_size());
+    return t;
+}
+
+template <typename T>
+Tensor make_tensor_of_value(const element::Type_t& et, const T& value) {
+    const ov::Shape& scalar_shape = {};
+    const std::vector<T>& value_vector = {value};
+    return make_tensor_of_value(et, scalar_shape, value_vector);
+}
+
+template <typename T>
+Tensor ge(const ov::Tensor& lhs, const T& element) {
+    if (!lhs)
+        return {};
+    const auto& other = make_tensor_of_value(lhs.get_element_type(), element);
+    return ge(lhs, other);
+}
+
+template <typename T>
+std::vector<T> to_vector(const ov::Tensor& t) {
```

Comment:
Is it required?
In most cases the tensor can be check (it can work like optional has got data or not). Then if initialized Constant can be used locally. return empty vector will also required some check and empty vector can not mean as not initialized tensor.